src/sentry/static/sentry/app/views/organizationDiscover/result/pagination.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import styled from 'react-emotion';
+import {Flex} from 'grid-emotion';
+
+import {t, tct} from 'app/locale';
+import Button from 'app/components/button';
+
+export default class Result extends React.Component {
+
+  getNextPage() {
+    this.props.onFetchPage('next');
```

Comment:
Oh, sorry I missed that since there wasn't a proptypes definition in this file. Might be a bit clearer to have the Pagination component just get `fetchNextPage` and `fetchPrevPage` as props, and not have these functions in here.